manifest.yml

```diff
@@ -1,6 +1,6 @@
 ---
 declared-services:
-  my-discovery-service:
+  discovery-breaches-service:
```

Comment:
I've been trying to use <prefix>-discovery-service so we look consistent w/ prefixes like wos- wbc- etc... Have seen that used in other examples too, but don't really know if we have been consistent enough to say that is THE pattern to follow.